requirements.txt

```diff
@@ -57,5 +57,8 @@ supervisor==3.3.1
 tzlocal==1.4
 uritemplate==0.6
 urllib3==1.22
-uwsgi==2.0.12
+uwsgi==2.0.17.1
 virtualenv==1.11.4
+moto==1.3.6
+WebTest==2.0.32
+freezegun==0.3.11
```

Comment:
Webtest and freezegun are only needed for development purposes (running tests), since this is still specified in tests/requirements_tests.txt, can we remove it from this?

Reply:
Looks like these were added by @zsck probably good to remove them here.